File: workspace/data-model/src/main/java/de/pfiva/data/model/NLUData.java
package de.pfiva.data.model;

import java.io.Serializable;

import de.pfiva.data.model.snips.SnipsOutput;

public class NLUData implements Serializable {

	private static final long serialVersionUID = 1L;
	
	private SnipsOutput snipsOutput;
	private Feedback feedback;
	private User user;
	
	public SnipsOutput getSnipsOutput() {
		return snipsOutput;
	}
	public void setSnipsOutput(SnipsOutput snipsOutput) {
		this.snipsOutput = snipsOutput;
	}
	public Feedback getFeedback() {
		return feedback;
	}
	public void setFeedback(Feedback feedback) {
		this.feedback = feedback;
	}
	public User getUser() {
		return user;
	}
	public void setUser(User user) {
		this.user = user;
	}
	@Override
	public String toString() {
		return "NLUData [snipsOutput=" + snipsOutput + ", feedback=" + feedback + ", user=" + user + "]";
	}
}

File: angular_workspace/pfiva-web/src/app/data-model/NLUData.ts
import { Feedback } from "./Feedback";
import { SnipsOutput } from "./snips/SnipsOutput";
import { User } from "./User";

export class NLUData {

    private _feedback: Feedback;
	private _snipsOutput: SnipsOutput;
	private _user: User;

	public get feedback(): Feedback {
		return this._feedback;
	}

	public set feedback(value: Feedback) {
		this._feedback = value;
    }
    
	public get snipsOutput(): SnipsOutput {
		return this._snipsOutput;
	}

	public set snipsOutput(value: SnipsOutput) {
		this._snipsOutput = value;
	}

	public get user(): User {
		return this._user;
	}

	public set user(value: User) {
		this._user = value;
	}
}
